src/terminal_view/Renderer.cpp

```diff
@@ -170,7 +170,7 @@ uint64_t Renderer::render(Terminal& _terminal,
         else
         {
             changes = _terminal.preRender(_now);
-            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, Cell const& _cell) { renderCell(_row, _col, _cell); },
+            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, terminal::ScreenBuffer::Cell const& _cell) { renderCell(_row, _col, _cell); },
```

Comment:
```suggestion
            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, ScreenBuffer::Cell const& _cell) { renderCell(_row, _col, _cell); },
```

Reply:
that looks right. it builds fine without the namespace 'terminal'.